include/spirv-tools/libspirv.h

```diff
@@ -375,6 +379,21 @@ spv_context spvContextCreate(spv_target_env env);
 // Destroys the given context object.
 void spvContextDestroy(spv_context context);
 
+// Creates a Validator options object with default options.
+spv_validator_options spvValidatorOptionsCreate();
+
+// Destroys the given Validator options object.
+void spvValidatorOptionsDestroy(spv_validator_options options);
+
+// Records the maximum number of structure members that is considered valid in
```

Comment:
My intention was to have the spv_validator_options structure that holds _all_ command line arguments (not just limits). Therefore, there would be no need to create any new entry point.

[Any new command line option can be added as a member of the struct.](https://github.com/KhronosGroup/SPIRV-Tools/pull/558/files/6b36d00538394091b5f8f11041948576bf2358f3#diff-99eab7706bbdca4c52fcc674bd6710a3R113)

Does that make sense? or did I miss your point?